pkg/agent/core/ngt/config/config.go

```diff
@@ -46,7 +48,7 @@ func NewConfig(path string) (cfg *Data, err error) {
 	}
 
 	if cfg != nil {
-		cfg.Bind()
```

Comment:
Can  you tell me why do you remove the cfg.Bind() here?

Reply:
sorry. this is my misstake.
I think `cfg.Bind()` is correct :bow:

Reply:
fixed: https://github.com/vdaas/vald/pull/1219/commits/164baadf6da88821fcb5eed9196dae2cd0c11a77